Working Mode.md

```diff
@@ -32,6 +32,14 @@ The editor has the following responsibilities, all of which can be delegated to
    * Only add features that have sufficient implementer interest.
    * Align existing features with implementations (e.g., via removal if implementer interest waned).
 
+## Meetings
```

Comment:
Given that we don't generally do them, perhaps we should move this down, e.g., before "Anchor permanence"?